CRM/Contact/Form/Edit/Notes.php

```diff
@@ -42,8 +42,8 @@ class CRM_Contact_Form_Edit_Notes {
    */
   public static function buildQuickForm(&$form) {
     $form->applyFilter('__ALL__', 'trim');
-    $form->add('text', 'subject', ts('Subject'), array('size' => 60, 'maxlength' => 254));
-    $form->add('textarea', 'note', ts('Notes'), array('cols' => '60', 'rows' => '3'));
+    $form->addField('subject', array('entity' => 'note', 'size' => 60, 'maxlength' => 254));
```

Comment:
Can you try leaving out the size & maxlength attributes? I'm pretty sure these are in the schema.